streamer/src/quic.rs

```diff
@@ -177,6 +177,10 @@ pub struct StreamStats {
     pub(crate) stream_load_capacity_overflow: AtomicUsize,
     pub(crate) process_sampled_packets_us_hist: Mutex<histogram::Histogram>,
     pub(crate) perf_track_overhead_us: AtomicU64,
+    pub(crate) total_staked_packets_sent_for_batching: AtomicUsize,
+    pub(crate) total_unstaked_packets_sent_for_batching: AtomicUsize,
+    pub(crate) throttled_staked_streams: AtomicUsize,
+    pub(crate) throttled_unstaked_streams: AtomicUsize,
```

Comment:
Change looks good to me; should we deprecate `total_packets_sent_for_batching` and `throttled_streams`; both of these can now be implied by adding unstaked + staked

Reply:
I think it is okay to leave it alone for convenience in doing Grafana.